lib/MusicBrainz/Server/Data/Relationship.pm

```diff
@@ -135,14 +135,22 @@ sub _load
         my (@cond, @params, $target, $target_id, $source_id, $query);
 
         if ($type eq $type0) {
-            push @cond, "entity0 IN (" . placeholders(@ids) . ")";
+            my $condstring = "entity0 IN (" . placeholders(@ids) . ")";
+            if ($use_cardinality) {
+                $condstring = '(' . $condstring . ' AND entity0_cardinality = 0)'
```

Comment:
There's a semicolon missing at the end of the line (and I think I'd just use double quotes here instead of concatenation, though it doesn't particularly matter other than saving characters).